.github/workflows/_rocm-test.yml

```diff
@@ -62,6 +62,15 @@ on:
         required: false
         type: number
         default: 1
+    secrets:
+      HUGGING_FACE_HUB_TOKEN:
+        required: false
+        description: |
+          HF Auth token to avoid rate limits when downloading models or datasets from hub
+      VLLM_TEST_HUGGING_FACE_TOKEN:
```

Comment:
You don't need `VLLM_TEST_HUGGING_FACE_TOKEN` here, maybe this can be needed later once we run vLLM tests on ROCm runners